R/shiny.R

```diff
@@ -1457,7 +1457,7 @@ ShinySession <- R6Class(
       # that string isn't a valid CSS color, will return NA)
       # https://github.com/rstudio/htmltools/issues/161
       parse_css_colors <- function(x) {
-        htmltools::parseCssColors(x %OR% "", mustWork = FALSE)
+        htmltools::parseCssColors(x %||% "", mustWork = FALSE)
```

Comment:
I can't remember if it's actually possible for `x` to have `NA`s, but we considered that possibility when writing this code (read the comment above). Should we just fix this in htmltools? https://github.com/rstudio/htmltools/issues/161

Reply:
Here's a PR to make `parseCssColors()` better handle `NA` values  https://github.com/rstudio/htmltools/pull/194

Reply:
Thanks, good catch!